documentation/src/main/asciidoc/topics/proc_registering_sci_embedded_caches.adoc

```diff
@@ -1,11 +1,13 @@
-[id='manually_registering_sci-{context}']
-= Manually Registering Serialization Context Initializers
+[id='registering-sci-embedded-caches_{context}']
+= Manually registering serialization context initializers
```

Comment:
As we explain that SCIs are automatically registered by default, as well as the manual process, maybe this section should just be "Registering Serialization Context Initializers"?

Reply:
makes sense